jobs/signing/sign-artifacts/Jenkinsfile

```diff
@@ -363,7 +377,13 @@ node {
                                 sh "aws s3 sync --no-progress --exclude='*' --include 'sha256=*' ./ s3://art-srv-enterprise/pub/rhacs/signatures/rh-acs/"
                             }
                         }
-
+                    } else if ( params.PRODUCT == 'coreos-installer' ) {
+                        sshagent(["openshift-bot"]) {
+                            withCredentials([aws(credentialsId: 's3-art-srv-enterprise', accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY')]) {
+                                sh "aws s3 sync --no-progress --exclude='*' --include 'sha256sum.txt.gpg' ./ s3://art-srv-enterprise/pub/openshift-v4/${params.ARCH}/clients/coreos-installer/${params.NAME}/"
```

Comment:
This is not where currently coreos-installer is: https://mirror.openshift.com/pub/openshift-v4/s390x/clients/

It is directly under clients, the separation of different arches is in the folder for the version: https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/v0.13.1-1/

Reply:
we only need push clients to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/coreos-installer/